packages/components/tests/acceptance/percy-test.js

```diff
@@ -2,10 +2,17 @@ import { module, test } from 'qunit';
 import { visit } from '@ember/test-helpers';
 import { setupApplicationTest } from 'ember-qunit';
 import percySnapshot from '@percy/ember';
+import config from 'dummy/config/environment';
 
 module('Acceptance | Percy test', function (hooks) {
   setupApplicationTest(hooks);
 
+  if (config.emberTryScenario) {
+    // eslint-disable-next-line no-console
+    console.log('Not running percy in ember-try');
+    return;
+  }
```

Comment:
I think we don't need this in flight since the ember-try isn't present in `packages/flight-website`